app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Your code says `Comment` belongs to `user` but your migration file for creating `comments` references `author_id`. I would advise you to reference `author` in your migration and then in your model file, `Comment` should belong to `author` instead of `user`. I won't be able to create a comment if this change is not made and that is the reason why I am making it a required change. 👍